api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -324,7 +324,14 @@ private DataSetRequest generateDataSetRequestFromPreviewRequest(
     DataSetPreviewResponse previewQueryResponse = new DataSetPreviewResponse();
     DataSetRequest dataSetRequest = generateDataSetRequestFromPreviewRequest(dataSetPreviewRequest);
     Map<String, QueryJobConfiguration> bigQueryJobConfig =
-        dataSetService.generateQueryJobConfigurationsByDomainName(dataSetRequest);
+        dataSetService.generateQueryJobConfigurationsByDomainName(dataSetRequest).entrySet()
+            .stream()
+            .collect(
+                Collectors.toMap(
+                    Map.Entry::getKey,
+                    stringQueryJobConfigurationEntry ->
+                        bigQueryService.filterBigQueryConfig(
```

Comment:
This deserves a comment, it seems like you're trying to replace the `{project}.{dataset}` templates within the query parameters at this step. But I'm not very clear on why

Reply:
I tried checking this and it appears to be nonsense, so I removed it